src/main/java/frc/robot/commands/ToggleDriveTarget.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.DrivetrainSubsystem;
+
+public class ToggleDriveTarget extends CommandBase{
```

Comment:
The name for this command is better as SetDriveRotationTarget now that it is no longer toggling.